onmt/decoders/decoder.py

```diff
@@ -28,7 +28,7 @@ class RNNDecoderBase(nn.Module):
           G[Decoder State]
           H[Decoder State]
           I[Outputs]
-          F[Memory_Bank]
+          F[attn_context]
```

Comment:
Not sure why this change of terminology is part of this PR. In addition to that, I don't think it's a good change: context usally refers to the output of the attention layer and not the output of the encoder. I strongly suggest reverting this renaming from this PR.